### DISCUSSION TITLE
Port 25 blocked by ComcastXfinity ISP. Can use different port?

### DISCUSSION THREAD

**Discussion by whisperdancer** [simple-login/app#1296] [Self-hosting question]
I noticed in your Readme.md, under prerequisites:

 The server needs to have the port 25 (email)

Email is used for important communications and Comcast wants to ensure that these communications are as secure and as private as possible. As such, Comcast does not support port 25 for the transmission of email by our residential Internet customers. Much of the current use of port 25 is by computers that have been infected by malware and are sending spam without the knowledge of the users of those computers.

https://www.xfinity.com/support/articles/list-of-blocked-ports

I surmise this may be why I can't get self hosting to work. Is it possible in configuration files to change the port to 587 which is open and secure TLS port?

**Comment by nguyenkims** [comment]
@whisperdancer the port 25 is used by other email services when they send emails to your server so not using it will lead to undelivered emails. We recommend to self host SimpleLogin on a cloud service and not using your personal computer as the port 25 is often blocked by ISP.
